source/_components/somfy_mylink.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "Somfy MyLink"
+description: "Instructions on how to integrate Somfy MyLink devices with Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Hub
+ha_release: 0.81
+ha_iot_class: "Assumed State"
+---
+
+The `Somfy MyLink` component platform is used as an interface to a compatible Somfy MyLink hub utilizing the `Synergy` API. It adds covers and scenes from the Somfy MyLink platform.
+
+To use your compatible `Somfy MyLink` devices in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+somfy_mylink:
+  host: 10.1.1.100
+  password: mylink_id
+```
+
+```yaml
```

Comment:
Please move the second example, after the configuration list.

Reply:
@frenck, I've moved this. Sorry for the delay, the GitHub outage has been playing havoc with my commits.